package/CHANGELOG

```diff
@@ -40,6 +40,7 @@ Enhancements
     option from setup.cfg (Issue #3428, PR #3429).
   * Added R/S Chirality support to RDKit parsed molecules
   * Added MDAnalysis.lib.distances.minimize_vectors
+  * Supports TPR files produced by Gromacs 2022 (PR #3514)
```

Comment:
tsk tsk tsk entries should be newest first

Reply:
You can tell I haven't pushed for a while :sweat_smile: